src/platform/qt/ts/mgba-zh_CN.ts

```diff
@@ -832,72 +827,77 @@ Game Boy Advance 是任天堂有限公司（Nintendo Co., Ltd.）的注册商标
         <translation>检查地址:</translation>
     </message>
     <message>
-        <location filename="../MemoryView.ui" line="61"/>
+        <location filename="../MemoryView.ui" line="58"/>
+        <source>:</source>
+        <translation>:</translation>
+    </message>
+    <message>
+        <location filename="../MemoryView.ui" line="68"/>
         <source>0x</source>
         <translation>0</translation>
```

Comment:
I should probably remove this string, but I'm pretty sure this should stay as 0x--it's a hexadecimal prefix marker.